Library/Homebrew/cask/cmd/info.rb

```diff
@@ -25,52 +26,64 @@ def self.help
         "displays information about the given Cask"
       end
 
+      def self.get_info(cask)
+        <<~EOS.chomp
+          #{title_info(cask)}
+          #{Formatter.url(cask.homepage) if cask.homepage}
```

Comment:
This will make a newline if there is no Cask homepage. I think the previous version was probably better but just use it to keep adding to a string.

Reply:
Okay, you mean instead of using a multiline string with interpolation, just append the return values of each function to a value that gets returned by get_info?